server_test.go

```diff
@@ -1008,6 +1008,39 @@ func TestServer_ExecuteQuery(t *testing.T) {
 	}
 }
 
+// Ensure that when querying for raw data values that they return in time order
+func TestServer_RawDataReturnsInOrder(t *testing.T) {
+	s := OpenServer(NewMessagingClient())
+	defer s.Close()
+	s.CreateDatabase("foo")
+	s.CreateRetentionPolicy("foo", &influxdb.RetentionPolicy{Name: "raw", Duration: 1 * time.Hour})
+	s.SetDefaultRetentionPolicy("foo", "raw")
+	s.CreateUser("susy", "pass", false)
```

Comment:
FWIW, we don't need to create "susy". We should remove this from the other tests too.